### PR TITLE
Merge branch release/8.8.x into develop (changelog, version number)

### PR DESCRIPTION
**Pull request by flomnes**
Updated changelog, version number

### CODE REVIEW

File: src/solver/variable/economy/STStorageCashFlowByCluster.h [hunk 1]
```diff
@@ -278,14 +278,16 @@ class STstorageCashFlowByCluster : public Variable::IVariable<STstorageCashFlowB
             const auto& shortTermStorage = results.data.area->shortTermStorage;
 
             // Write the data for the current year
-            for (uint clusterIndex = 0; clusterIndex < nbClusters_; ++clusterIndex)
+            uint clusterIndex = 0;
```
Comment (flomnes) [inline src/solver/variable/economy/STStorageCashFlowByCluster.h]: `clusterIndex` appears to be unused

File: src/solver/variable/economy/STStorageInjectionByCluster.h [hunk 2]
```diff
@@ -280,14 +280,16 @@ class STstorageInjectionByCluster : public Variable::IVariable<STstorageInjectio
             const auto& shortTermStorage = results.data.area->shortTermStorage;
 
             // Write the data for the current year
-            for (uint clusterIndex = 0; clusterIndex < nbClusters_; ++clusterIndex)
+            uint clusterIndex = 0;
```
Comment (flomnes) [inline src/solver/variable/economy/STStorageInjectionByCluster.h]: unused

File: src/solver/variable/economy/STStorageLevelsByCluster.h [hunk 3]
```diff
@@ -280,14 +280,16 @@ class STstorageLevelsByCluster
             const auto& shortTermStorage = results.data.area->shortTermStorage;
 
             // Write the data for the current year
-            for (uint clusterIndex = 0; clusterIndex < nbClusters_; ++clusterIndex)
+            uint clusterIndex = 0;
```
Comment (flomnes) [inline src/solver/variable/economy/STStorageLevelsByCluster.h]: unused

File: src/solver/variable/economy/STStorageWithdrawalByCluster.h [hunk 4]
```diff
@@ -280,14 +280,16 @@ class STstorageWithdrawalByCluster
             const auto& shortTermStorage = results.data.area->shortTermStorage;
 
             // Write the data for the current year
-            for (uint clusterIndex = 0; clusterIndex < nbClusters_; ++clusterIndex)
+            uint clusterIndex = 0;
```
Comment (flomnes) [inline src/solver/variable/economy/STStorageWithdrawalByCluster.h]: unused

File: docs/CHANGELOG.md [hunk 5]
```diff
@@ -6,32 +6,13 @@ Antares Changelog
...
-## Bugfixes
-* Prevent segfault during simulation, check bounds of scenario builder (#1567)
-* Fix number of links in deprecated output file digest.txt (#1646)
-* Fix unfeasible problem analyzer (#1527)
-* [Windows only] Increase file size limit when reading file (#1674)
-* Fix segfault encountered when importing logs (#1702)
-* Fixes swallowed exceptions in computation thread (#1685)
-* Fix writer causing a segfault with OR-Tools (#1584)
-* Bug on renewable cluster (wrong group) (#1631)
+* Experimental optimization with discrete variables (MILP unit-commitment mode, #670)
```
Comment (a-zakir) [inline docs/CHANGELOG.md]:
```suggestion
* Experimental optimization with discrete variables (MILP unit-commitment mode #670)
```

**Review by a-zakir — APPROVED** [review]

**Pull request merged**